lit_tests/kernel/wave/scaled_gemm.py

```diff
@@ -0,0 +1,430 @@
+# RUN: python %s | FileCheck %s
+
+import iree.turbine.kernel.lang as tkl
+import iree.turbine.kernel.wave as tkw
+from iree.turbine.kernel.wave.compile import WaveCompileOptions, wave_compile
+from iree.turbine.kernel.wave.scheduling.schedule_enums import SchedulingType
+
+from iree.turbine.kernel.lang.global_symbols import *
+from iree.turbine.kernel.wave.utils.general_utils import (
+    get_default_scheduling_params,
+    run_test,
+)
+from iree.turbine.kernel.wave.constraints import (
+    ScaledMMAType,
+)
+
+# Note this is specified by the HW and cannot be changed.
+SCALE_GROUP_SIZE = 32
+
+
+@run_test
+def test_scaled_gemm_mxfp4():
+    # Input sizes
```

Comment:
@tgymnich Just curious, what is the main difference between the scaled_gemm.py and scaled_mma.py, thoughts on making it into a single file?

Reply:
Understood, no action for now, but let me chat with @harsh-nod to see if we can eventually combine the even non-scaled version to a single file.